src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
Okay - look. I don't mind waiting for another person's input, but I have to at least ensure that we reach a consensus on the model of the situation. The following may include contrived examples, but work with me here:

1. Running `gcc.exe $FILENAME_HERE` is going to result in `gcc` trying to compile the file in both PowerShell and MSYS2 `bash`.
2. Running `notepad.exe $FILENAME_HERE` is going to result in `notepad.exe` trying to open the file.
3. Running `gcc @$FILENAME_HERE` is going to result in `gcc` interpreting `$FILENAME_HERE` as a command file on MSYS2 `bash`, and PowerShell getting confused about the splatting operator.
4. Running `notepad.exe @$FILENAME_HERE` is going to result in PowerShell wondering why you want to use the splatting operator, and in `bash` it'll try to open the literal file `"@$FILENAME_HERE"`.

Where are you getting that this is standardized, and even if it is how does that connect to our expected use of `gcc` (and `clang`, and maybe eventually `ccache gcc` or `ccache clang`) on MSYS2 `bash`?